CHANGELOG.md

```diff
@@ -67,6 +68,7 @@ Known issues:
 - Fixed crash caused by MGLAnnotationImage with non-integer width or height ([#2198](https://github.com/mapbox/mapbox-gl-native/issues/2198))
 - Fixed “include of non-modular header” errors in Swift projects managed by CocoaPods. ([#3679](https://github.com/mapbox/mapbox-gl-native/pull/3679)) 
 - Avoids triggering the blue background location status bar when user has granted "when in use" permission. ([#3671](https://github.com/mapbox/mapbox-gl-native/issues/3671))
+- Added new methods to MGLMapView for getting and setting the minimum and maximum zoom levels: `minimumZoomLevel`, `setMinimumZoomLevel`, `maximumZoomLevel`, `setMaximumZoomLevel`. ([#509](https://github.com/mapbox/mapbox-gl-native/issues/509))
```

Comment:
What’s the default in GL JS? [22](https://github.com/mapbox/mapbox-gl-js/blob/739aa39e340f523fd9b7e431a0bb366b41a91711/js/geo/transform.js#L28)? Are the issues enumerated in https://github.com/mapbox/mapbox-gl-native/issues/509#issuecomment-61131373 still relevant?